frontends/README.md

```diff
@@ -0,0 +1,14 @@
+# frontends
+
+Experimental frontends for Sonobe.
+The recommended frontend is to directly use [arkworks](https://github.com/arkworks-rs) to define the FCircuit, just following the [`FCircuit` trait](https://github.com/privacy-scaling-explorations/sonobe/blob/main/folding-schemes/src/frontend/mod.rs).
+
+## Experimental frontends
+> Warning: the following frontends are experimental and some computational and time overhead is expected when using them compared to directly using the [arkworks frontend](https://github.com/privacy-scaling-explorations/sonobe/blob/main/folding-schemes/src/frontend/mod.rs).
+
+- [Circom](https://github.com/iden3/circom), iden3, 0Kims Association (supported v2.1.9)
+- [Noir](https://github.com/noir-lang/noir), Aztec
+- [Noname](https://github.com/zksecurity/noname), zkSecurity
```

Comment:
We should warn that noname is partially supported, as suggested in #150 . We can then also close this issue.

Reply:
Cool, added the mention ^^